data/anope.example.conf

```diff
@@ -134,22 +134,31 @@ define
  *     <uline server="services.localhost.net" silent="yes">
  *     <bind address="127.0.0.1" port="7000" type="servers">
  *
- * An example configuration for UnrealIRCd that is compatible with the below uplink
+ * An example configuration for UnrealIRCd 4.x that is compatible with the below uplink
  * and serverinfo configuration would look like:
  *
- *     link services.localhost.net
- *     {
- *          username *;
- *          hostname *;
- *          bind-ip "127.0.0.1";
- *          port 7000;
- *          hub *;
- *          password-connect "mypassword";
- *          password-receive "mypassword";
- *          class servers;
- *     };
- *     ulines { services.localhost.net; };
- *     listen 127.0.0.1:7000;
+ * listen {
+ *   ip 127.0.0.1;
+ *   port 7000;
+ *    options {
+ *     serversonly;
+ *    };
+ * };
+ *
+ * link services.localhost.net
+ * {
+ *   incoming {
+ *    mask *@127.0.0.1;
+ *   };
+ *   outgoing { 
```

Comment:
Is outgoing necessary in the link block? You can't /connect services

Reply:
And you're correct. For services the outgoing block isn't needed. I also went with `*@127.0.0.1` in the incoming because I add a few issues with connection denied because it said that anope@127.0.0.1 wasn't a valid link block or something like that.